metron-deployment/packaging/packer-build/.gitignore

```diff
@@ -1,6 +1,6 @@
 .bundle/
 iso
-*.box
+builds/base-centos-6.7-2.1.20170303223924.git.33abe8cf13c347a2dfdece145a7b8c17f2a423c0_dirty.virtualbox.box
```

Comment:
Urm, no. I must have dumped my copy buffer by mistake while IntelliJ was open. It'll be gone in the next changeset. Thanks!